lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -133,6 +135,23 @@ def test_credit_requirements_not_eligible(self):
         )
         self.assertContains(response, "Verification Failed")
 
+    @ddt.data('honor', 'audit', 'verified', 'credit')
+    def test_credit_requirements_non_credit_enrollment(self, enrollment_mode):
+        # Test the progress table is only displayed to the
+        # verified and credit students.
+        self.enrollment.mode = enrollment_mode
+        self.enrollment.save()  # pylint: disable=no-member
+
+        # Check the progress page display
+        response = self._get_progress_page()
+        if enrollment_mode in ['credit', 'verified']:
```

Comment:
this test suite is also already testing `verified`, so it's redundant to check for it.